lib/assert.js

```diff
@@ -60,11 +60,21 @@ function isSWEnv() {
     self instanceof ServiceWorkerGlobalScope);
 }
 
+function isValue(object, expectedValue) {
+  const parameter = Object.keys(object).pop();
```

Comment:
Wrapping in objects is more work for us when we write the library code, but I think it's worth the benefit offered to folks consuming our libraries. E.g., the pattern allows for a standard way of reporting incorrect library usage, like the following:

Assume the following assertions in our library code:

```js
assert.isType({cacheName}, 'string');
assert.atLeastOne({maxEntries, maxAgeSeconds});
```

Assume a third party developer has the following buggy usage:
```js
new goog.cacheExpiration.Behavior({
  cacheName,
});
```

That leads to `Please set at least one of the following parameters: 'maxEntries', 'maxAgeSeconds'` being logged.

Another example of buggy library usage:
```js
new goog.cacheExpiration.Behavior({
  maxEntries: 2,
  maxAgeSeconds: 10,
});
```

leads to `The 'cacheName' parameter has the wrong type. (Expected: string, actual: undefined)` being logged.

Wrapping the parameters passed to the assertions in an object is the easiest way I could think of to get the actual parameter names passed along to the assertion library, and third party developers get much more useful error messages as a result.

@gauntface, I'd love to fix whatever bugs were stopping you from using the assertions.